cogs/birthday/constants.py

```diff
@@ -20,6 +20,7 @@
     KEY_ALLOW_SELF_BDAY: False,
 }
 
+BOT_BIRTHDAY = "Wow! It's my own birthday! Happy birthday to myself!"
```

Comment:
Perhaps a better name could be used? I mean, it sounds more like this constant is about the bot's birthday than the message sent on the day.
```suggestion
BOT_BIRTHDAY_MSG = "Wow! It's my own birthday! Happy birthday to myself!"
```

Reply:
ack